SourceCgit/SourceCgit.php

```diff
@@ -238,7 +238,8 @@ private function import_commits( $p_repo, $p_uri_base, $p_commit_ids, $p_branch=
 	 * @return string the revision
 	 */
 	public function commit_revision( $p_input ) {
-		preg_match( "#<tr><th>commit</th><td colspan='2' class='sha1'><a href='/(.*?)/commit/\?id=([a-f0-9]*)'>([a-f0-9]*)</a>#", $p_input, $t_matches);
+		$patter = '#<tr><th>commit</th><td colspan=\'2\' class=\'oid\'><a href=\'/(.*?)/commit/\?id=([a-f0-9]*)\'>([a-f0-9]*)</a>#';
```

Comment:
I'm fine with extracting the regex into a variable, but why change from a double-quoted to single-quoted string ? This introduces escaping of single quotes, which makes the regex more difficult to read IMO

Variable name should be `$pattern` not `$patter` I guess

Reply:
good question, I don't remember why I've wrote that. And `$patter` is just a typo